README.md

```diff
@@ -129,9 +129,7 @@ const mongoose = require("mongoose");
 const { driver, createAstraUri } = require("stargate-mongoose");
 
 const uri = createAstraUri(
-  process.env.ASTRA_DB_ID,
-  process.env.ASTRA_DB_REGION,
-  process.env.ASTRA_DB_KEYSPACE,
+  process.env.ASTRA_DB_API_ENDPOINT,
   process.env.ASTRA_DB_APPLICATION_TOKEN,
```

Comment:
Should also add `ASTRA_DB_NAMESPACE` as 3rd param

Reply:
I think its an optional param and in Astra, the default namespace that gets created is `default_keyspace` which is connected to, by this code by default if we don't provide one explicitly.

Reply:
I know it's optional, but I think it is worthwhile to provide an example of how to set the URI's namespace. You can add `// optional` to indicate that the namespace argument is optional, but I think it is worth having.

Reply:
Sure, Fixed in this commit [4152374](https://github.com/stargate/stargate-mongoose/pull/179/commits/415237494f9c3d8e1d1175234ed6ecc0f5a0eebd).